tests/test_create_agent.py

```
@@ -0,0 +1,623 @@
+import pytest
+
+from conftest import setup_temp_project
+
+# Get project root for script paths
+PROJECT_ROOT = Path(__file__).parent.parent
+SCRIPT_PATH = PROJECT_ROOT / "scripts" / "create-agent.sh"
+TEMPLATE_PATH = PROJECT_ROOT / ".claude" / "agents" / "AGENT-TEMPLATE.md"
+LAUNCHER_PATH = PROJECT_ROOT / "agents" / "launch"
+
+
+LOCK_FILE = Path("/tmp/agent-creation-launcher.lock")
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/agent-creation-launcher.lock"

(S108)